tabular/src/autogluon/tabular/learner/default_learner.py

```diff
@@ -32,12 +32,14 @@ def __init__(self, trainer_type=AutoTrainer, **kwargs):
         self._time_limit = None
 
     # TODO: Add trainer_kwargs to simplify parameter count and extensibility
-    def _fit(self, X: DataFrame, X_val: DataFrame = None, scheduler_options=None, hyperparameter_tune=False,
+
+    def _fit(self, X: DataFrame, X_val: DataFrame = None, X_unlabeled: DataFrame = None, scheduler_options=None, hyperparameter_tune=False,
             feature_prune=False, holdout_frac=0.1, num_bagging_folds=0, num_bagging_sets=1, stack_ensemble_levels=0,
             hyperparameters=None, ag_args_fit=None, excluded_model_types=None, time_limit=None, save_data=False, save_bagged_folds=True, verbosity=2):
         """ Arguments:
                 X (DataFrame): training data
                 X_val (DataFrame): data used for hyperparameter tuning. Note: final model may be trained using this data as well as training data
+                X_unlabeled (DataFrame): data used for pretraining a model. Mostly used in semi-supervised situations.
```

Comment:
clarify this is same data format as X, except label-column missing

Reply:
Changed comment.